src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -232,6 +232,22 @@
   <data name="Globals_LaunchModeMaximized.Content" xml:space="preserve">
     <value>Maximized</value>
   </data>
+  <data name="Globals_WindowingBehavior.Header" xml:space="preserve">
+    <value>New windows open in...</value>

```

Comment:
```suggestion
    <value>wt.exe window behavior</value>
```

Reply:
Hmmmmm... this will be weird when Def Term is in

Reply:
@cinnamon-msft Yea I concur. Plus I'm just not sure that the users will always think "ah yes, I'm running `wt.exe`". Like when running from the start menu - that's _also_ going to behave this way. 

I'm fine flipping these back since what's proposed is basically what I had originally - I just want to make sure we're _sure_, and I'm not so sure about mentioning "wt.exe"

Reply:
Maybe `Launch window behavior`?